README.md

```diff
@@ -430,15 +430,15 @@ To update `angular-cli` to a new version, you must update both the global packag
 
 Global package:
 ```bash
-npm uninstall -g angular-cli
+npm uninstall -g angular-cli or npm uninstall -g @angular/cli
```

Comment:
Let's keep `angular-cli` a separate line with a bash comment saying something like "for old versions < beta 28.x" or so.